### PR TITLE
Using LabVIEW Dashboard with C++/Java code needs updates

### LINKED ISSUE #984: Using LabVIEW Dashboard with C++/Java code needs updates
C++ SmartDashboard calls need frc namespace
There's references that SmartDashboard classes don't support arrays, but they have for 4 years, so examples should be added.

### PR DESCRIPTION
**Pull request by jasondaming**
Closes #984

### CODE REVIEW

**Comment by Daltz333** [comment]
@calcmogul or @prateekma the java code looks fine to me, can you verify the C++

File: source/docs/software/labview-dashboard/using-the-labview-dashboard-with-c++-java-code.rst [hunk 1]
```diff
@@ -8,7 +8,52 @@ Drive Tab
...
+            // auto here
+          case "Drive Backwards":
+            // auto here
+          case "Shoot":
+            // auto here
+       }
+
+    .. code-tab:: c++
+
+       frc::SmartDashboard::PutNumber("Gyro", Drivetrain.GetHeading());
```
Comment (calcmogul) [inline source/docs/software/labview-dashboard/using-the-labview-dashboard-with-c++-java-code.rst:32]: These C++ and Java snippets should do the same thing but don't.

**Review by calcmogul — CHANGES_REQUESTED** [review]

File: source/docs/software/labview-dashboard/using-the-labview-dashboard-with-c++-java-code.rst (hunk 1 above)
Comment (jasondaming) [reply source/docs/software/labview-dashboard/using-the-labview-dashboard-with-c++-java-code.rst]: Let me know if this looks good!

File: source/docs/software/labview-dashboard/using-the-labview-dashboard-with-c++-java-code.rst [hunk 2]
```diff
@@ -8,7 +8,63 @@ Drive Tab
...
 .. image:: images/using-the-labview-dashboard-with-c++-java-code/drive-tab.png
 
-Most of the indicators on the Drive tab utilize arrays.
+The :guilabel:`Select Autonomous...` dropdown can be used so show the available autonomous routines and choose one to run for the match.
+
+.. tabs::
+
+    .. code-tab:: java
+
+       SmartDashboard.putStringArray("Auto List", ["Drive Forwards", "Drive Backwards", "Shoot"]);
```
Comment (calcmogul) [inline source/docs/software/labview-dashboard/using-the-labview-dashboard-with-c++-java-code.rst:17]: Lists don't use square braces in Java. I don't know the correct way to do this though.

File: source/docs/software/labview-dashboard/using-the-labview-dashboard-with-c++-java-code.rst [hunk 3]
```diff
@@ -8,7 +8,63 @@ Drive Tab
...
+            // auto here
+          case "Drive Backwards":
+            // auto here
+          case "Shoot":
+            // auto here
+       }
+
+    .. code-tab:: c++
+
+       frc::SmartDashboard::PutStringArray("Auto List", ["Drive Forwards", "Drive Backwards", "Shoot"]);
```
Comment (calcmogul) [inline source/docs/software/labview-dashboard/using-the-labview-dashboard-with-c++-java-code.rst:32]:
```suggestion
       frc::SmartDashboard::PutStringArray("Auto List", {"Drive Forwards", "Drive Backwards", "Shoot"});
```

File: source/docs/software/labview-dashboard/using-the-labview-dashboard-with-c++-java-code.rst [hunk 4]
```diff
@@ -8,7 +8,63 @@ Drive Tab
...
+
+       frc::SmartDashboard::PutNumber("Gyro", Drivetrain.GetHeading());
+
+There are four outputs that show the motor power to the drivetrain.  This is configured for 2 motors per side and a tank style drivetrain.  This is done by setting "RobotDrive Motors" like the example below.
+
+.. tabs::
+
+    .. code-tab:: java
+
+       SmartDashboard.putNumberArray("RobotDrive Motors", [drivetrain.getLeftFront(), drivetrain.getRightFront(), drivetrain.getLeftBack(), drivetrain.getRightBack()]);
```
Comment (calcmogul) [inline source/docs/software/labview-dashboard/using-the-labview-dashboard-with-c++-java-code.rst:63]: Replace square braces with proper Java list syntax (`List.of()`? idk).

File: source/docs/software/labview-dashboard/using-the-labview-dashboard-with-c++-java-code.rst [hunk 5]
```diff
@@ -8,7 +8,63 @@ Drive Tab
...
+
+    .. code-tab:: c++
+
+       frc::SmartDashboard::PutNumberArray("Gyro", [drivetrain.GetLeftFront(), drivetrain.GetRightFront(), drivetrain.GetLeftBack(), drivetrain.GetRightBack()]);
```
Comment (calcmogul) [inline source/docs/software/labview-dashboard/using-the-labview-dashboard-with-c++-java-code.rst:67]:
```suggestion
       frc::SmartDashboard::PutNumberArray("Gyro", {drivetrain.GetLeftFront(), drivetrain.GetRightFront(), drivetrain.GetLeftBack(), drivetrain.GetRightBack()});
```

**Review by calcmogul — CHANGES_REQUESTED** [review]

**Comment by jasondaming** [comment]
@calcmogul it looks like Java can use {} for [arrays](https://www.tutorialspoint.com/java/java_arrays.htm#:~:text=Advertisements,variables%20of%20the%20same%20type.) too!

**Review by calcmogul — APPROVED** [review]

**Review by Daltz333 — APPROVED** [review]

**Pull request merged**